### PR TITLE
APIDOC-197 attempt to remove sidenav from tutorials.'

### PR DESCRIPTION
**Pull request by lggg123**
## Description

remove the sidenav from tutorials page.

### CODE REVIEW

File: lib/nexmo_developer/app/views/layouts/documentation.html.erb [hunk 1]
```diff
@@ -17,18 +17,30 @@
...
+      </main>
+    <%else%>
+      <main class="Vlt-main Nxd-main">
+        <%= render partial: 'layouts/partials/notices' %>
+        <div class="Vlt-grid">
+          <div class="Vlt-col Vlt-col--3of4 Nxd-tutorial">
+            <%= yield %>
+            <%= render partial: 'layouts/partials/improve_this_page' %>
+          </div>
+            <div class="Vlt-col Vlt-col--1of4 Vlt-sidenav Nxd-tutorial-sidenav">
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/layouts/documentation.html.erb:38]: wrap this in if/else using `controller_name` rails helper and remove @not_show_nav

**Comment by marcoranieri** [comment]
Almost there! Still some bug in `tutorials` page 🐛 

https://station-pr-1569.herokuapp.com/messages/tutorials/send-fbm-message/introduction

image

**Comment by lggg123** [comment]
> Almost there! Still some bug in `tutorials` page 🐛
> 
> https://station-pr-1569.herokuapp.com/messages/tutorials/send-fbm-message/introduction
> 
> image

looks like the unless statement I added isnt working.

**Comment by marcoranieri** [comment]
> > Almost there! Still some bug in `tutorials` page 🐛
> > https://station-pr-1569.herokuapp.com/messages/tutorials/send-fbm-message/introduction
> > image
> 
> looks like the unless statement I added isnt working.



I think `current_page?` method is more about the action than the controller (it's working with actions, params, show pages etc etc). 

Please use `controller_name` view helper for the IF statement

FYI I've pulled your branch and used a `raise` to check the problem
image

File: lib/nexmo_developer/app/views/layouts/documentation.html.erb [hunk 2]
```diff
@@ -17,18 +17,30 @@
         </nav>
       </div>
     </div>
-    <main class="Vlt-main Nxd-main">
-      <%= render partial: 'layouts/partials/notices' %>
-      <div class="Vlt-grid">
-        <div class="Vlt-col Vlt-col--3of4 Nxd-tutorial">
-          <%= yield %>
-          <%= render partial: 'layouts/partials/improve_this_page' %>
+    <% if controller.controller_name == 'tutorial' %>
```
Comment (marcoranieri) [inline lib/nexmo_developer/app/views/layouts/documentation.html.erb:20]: What do you think about this 🙂

```
    <main class="Vlt-main Nxd-main">
      <%= render partial: 'layouts/partials/notices' %>
      <div class="Vlt-grid">
      <% if navigation_from_content(content: @content).match?("<li>") && controller_name != 'tutorial' %>
        <div class="Vlt-col Vlt-col--3of4 Nxd-tutorial">
          <%= yield %>
          <%= render partial: 'layouts/partials/improve_this_page' %>
        </div>
        <div class="Vlt-col Vlt-col--1of4 Vlt-sidenav Nxd-tutorial-sidenav">
          <%= navigation_from_content(content: @content) %>
      <% else %>
        <div class="Vlt-col Vlt-col--5of4 Nxd-tutorial">
          <%= yield %>
          <%= render partial: 'layouts/partials/improve_this_page' %>
      <% end %>
        </div>
      </div>
    </main>

**Review by marcoranieri — APPROVED** [review]

**Pull request merged**